packages/tooltip/src/Tooltip.tsx

```diff
@@ -11,26 +11,57 @@ import { useMergeRefs } from '@lidofinance/hooks'
 import { isElement } from 'react-is'
 import { TooltipProps } from './types'
 
+const KEEP_TIMEOUT_DEFAULT = 150
+
 function Tooltip(props: TooltipProps, ref?: ForwardedRef<HTMLDivElement>) {
-  const { children, title, ...rest } = props
+  const {
+    title,
+    keepWhileHovering,
+    keepTimeout = KEEP_TIMEOUT_DEFAULT,
+    children,
+    ...rest
+  } = props
+
   const [state, setState] = useState(false)
+  const keepTimeoutRef = useRef(-1)
 
   const child = Children.only(children)
   if (!isElement(child)) throw new Error('Child must be a React element')
 
   const anchorRef = useRef<HTMLElement>(null)
   const mergedRef = useMergeRefs([child.ref, anchorRef])
 
+  const handleMouseEnterKeeping = () => {
+    if (keepTimeoutRef.current !== -1) {
```

Comment:
and here you can check for the existence of a timeout instead of checking for `-1`

Reply:
Ok, changed